app/blog/[id]/page.tsx

```
@@ -49,152 +123,232 @@ export default function BlogPost() {
+          </motion.div>
+
+          {/* Blog Content */}
+          <div className="container mx-auto px-4">
+            <div className="max-w-3xl mx-auto">
+              <motion.div
+                initial={{ opacity: 0, y: 20 }}
+                animate={{ opacity: 1, y: 0 }}
+                transition={{ duration: 0.6, delay: 0.2 }}
+                className="prose prose-invert prose-sm sm:prose-base md:prose-lg max-w-none"
+              >
+                <div dangerouslySetInnerHTML={{ __html: blog.content }} />
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)